angular-ui-router/angular-ui-router.d.ts

```diff
@@ -168,6 +168,17 @@ declare module angular.ui {
         current: IState;
         params: IStateParamsService;
         reload(): void;
+        
+        $current: IStateServiceUtilities;
+    }
+    
+    interface IStateServiceUtilities {
```

Comment:
Sounds fair to me; though, I won't update it to extend `IState`, since it doesn't appear to match. 

Do you think it's fair that I inlined the types for locals and globals? I really have no way to know what they are to name them.